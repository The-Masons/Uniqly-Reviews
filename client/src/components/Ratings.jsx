import React from 'react';
import ReactDOM from 'react-dom';
import axios from 'axios';

class Ratings extends React.Component {
    constructor(props) {
        super(props);
        this.state = {
        }
    }

    render() {
        return (
            <div className='review-ratings'>
                Review Ratings
            </div>
        );
    }
}

export default Ratings;
