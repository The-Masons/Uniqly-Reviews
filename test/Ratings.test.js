import React from 'react';
import ReactDOM from 'react-dom';
import Ratings from '../client/src/components/Ratings.jsx';
import renderer from 'react-test-renderer';

const sample = {
  product_id: 3,
  nickname: 'Alivia',
  from: 'West Israel, Indiana',
  size_purchased: 'L',
  age: '35 to 44,',
  gender: 'M',
  received_free_product: true,
  chest_size: 45,
  waist_size: 31,
  height: 'Over 6\'2',
  hips_size: 28,
  product_fit: 4,
  product_length: 1,
  product_quality: 2,
  date: '2017-12-24T20:14:40.054Z',
  rating: 4,
  review_title: 'pariatur quod occaecati',
  review_body: 'Earum necessitatibus animi doloribus aperiam. Ut enim alias accusamus.',
  recommended: true,
  helpful: 4,
  from_uk: false,
  verified_purchaser: true,
  employee: false,
  __v: 0 };

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Ratings reviewData={sample}/>, div);
});

describe('Ratings', () => {
  test('should render with the correct items', () => {
    const itemPicker = mount(
      <Ratings
        reviewData={sample}/>
      );
    expect(Ratings).toMatchSnapshot();
  });
});