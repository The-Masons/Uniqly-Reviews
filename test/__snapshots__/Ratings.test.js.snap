// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ratings should render with the correct items 1`] = `[Function]`;
